src/account/AccountService.java

```diff
@@ -39,11 +39,24 @@ public void deleteAccountById(int accountNumber) {
     }
 
     public void getAllAccounts() {
-            for (Account account : accountList){
-                System.out.println("Account Holders Name : " +account.accountHoldersName);
-                System.out.println("Account Number : " +account.accountNumber);
-                System.out.println("Account Balance : " +account.accountBalance);
-                System.out.println("Account Type : " +account.accountType);
+        for (Account account : accountList) {
+            System.out.println("Account Holders Name : " + account.accountHoldersName);
+            System.out.println("Account Number : " + account.accountNumber);
+            System.out.println("Account Balance : " + account.accountBalance);
+            System.out.println("Account Type : " + account.accountType);
+        }
+    }
+
+    public void getAccountByAccountNumber(int accountNumber) {
+        for (Account account : accountList) {
```

Comment:
you should use getAccountByAccounNumber method in here no need to write duplicate code